editors/nvim-lsp.nix

```diff
@@ -32,43 +32,43 @@ let
     -- Basic options --
     -------------------
     local options = {
-      clipboard = "unnamedplus",
-      mouse = "a",
-      undofile = true,
-      ignorecase = true,
-      showmode = false,
-      showtabline = 2,
-      smartindent = true,
-      autoindent = true,
-      swapfile = false,
-      hidden = true, --default on
-      expandtab = true,
-      cmdheight = 1,
-      shiftwidth = 2, --insert 2 spaces for each indentation
-      tabstop = 2, --insert 2 spaces for a tab
-      cursorline = true, --Highlight the line where the cursor is located
-      cursorcolumn = false,
-      number = true,
-      numberwidth = 4,
-      relativenumber = true,
-      scrolloff = 8,
-      updatetime = 50, -- faster completion (4000ms default)
+    	clipboard = "unnamedplus",
```

Comment:
Formatting changes necessary?

Reply:
There was a problem formatting the lua-related code in this pr #105 , so I reformatted it

Reply:
> There was a problem formatting the lua-related code in this pr https://github.com/nix-community/nixd/pull/105 , so I reformatted it

Ideally you should just push the main branch without review, to reformat your own code. Instead of submit PRs & request review. That might be some noise to the reviewer.